### DISCUSSION TITLE
"recognize:reset-tags" grows up database by 1.8GB

### DISCUSSION THREAD

**Discussion by UserXH726** [nextcloud/recognize#955] [Q&A] [answered]
Hello everyone,
I have here a (for me) strange behaviour by running the occ command recognize:reset-tags.
I run nextcloud 27.0.2 with php8.1, nginx and postgresql.

My goal was to make a clean reinstall of recognize.
So I executed the following occ commands:
- recognize:remove-legacy-tag
- recognize:cleanup-tags
- recognize:reset-tags
- recognize:reset-faces
- recognize:reset-face-clusters

While executing the command recognize:reset-tags I recognized that the size of my database was growing up by 2GBs.
After the commands are finished and I remove the app recognize, the database losts 200MBs.
So in sum the database growed up by 1.8GB.
Why is that so?

Is that a problem with the recognize:reset-faces command or is it postgresql problem?
I'm not an expert in php nor postgresql, so if you need more information, please describe me, how I can gather this for you.
Thank you

**Comment by marcelklehr** [comment, marked as answer]
Hey @UserXH726 

tags changes are reflected in the activity table, even when you remove all tags, nextcloud will create new entries in the activities table for all the removed tags.
